packages/flutter/lib/src/widgets/selectable_region.dart

```diff
@@ -533,6 +533,7 @@ class SelectableRegionState extends State<SelectableRegion> with TextSelectionDe
    } else {
      _selectionOverlay!.hideMagnifier();
      _selectionOverlay!.showToolbar(
+       context: context,
```

Comment:
Good catch, context should be provided when contextMenuBuilder is provided.  Thanks!